mmseg/datasets/transforms/transforms.py

```diff
@@ -1805,3 +1805,114 @@ def __repr__(self):
         repr_str += f'per_channel={self.per_channel},'
         repr_str += f'retain_stats={self.retain_stats}'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class BioMedical3DRandomFlip(BaseTransform):
+    """Flip biomedical 3D images and segmentations. Modified from
+    https://github.com/MIC-DKFZ/batchgenerators/blob/master/batchgenerators/tra
+    nsforms/spatial_transforms.py # noqa:E501 Copyright 2021 Division of
+    Medical Image Computing, German Cancer Research Center (DKFZ) and Applied
+    Computer Vision Lab, Helmholtz Imaging Platform. Licensed under the
+    Apache-2.0 License Required Keys:
+
+    - img
+    - gt_seg_map (optional)
+    Modified Keys:
+    - img
+    - gt_seg_map (optional))
+    Added Keys:
+    - do_flip
+    - flip_axes
+    Args:
+        prob (float): The flipping probability.
+        axes (Optional[Tuple[int, ...]]): A tuple of length no more than 3
+            and each element less than 3 is used as the rotation axis when
+            flipping.
+        swap_label_pairs (Optional[List[Tuple[int, int]]]): The labels of
+            some datasets need to be aligned after image and label flipping.
+            For example, the segmentation labels left hand and right hand
+            in the human parsing task in natural images need to be converted
+            after the images are flipped left and right.
+    """
+
+    def __init__(self,
+                 prob: float,
+                 axes: Optional[Tuple[int, ...]] = None,
```

Comment:
```suggestion
                 axes: Tuple[int, ...],
```
I think the axes cannot be None in this transform, and could you give an example about axes in docstring, like the order of axes is ZYX or ZXY?

Reply:
Okay, the order 'ZXY' is given in docstring.